### DISCUSSION TITLE
Apple Watch app

### DISCUSSION THREAD

**Discussion by SecrestTheHuman** [LeoKlaus/plappa#160] [Feature Requests]
I know this is on the to do list, but for whatever it’s worth I just wanted to voice my support for the feature in case that moves the needle on when it might be implemented. I’ve started leaving the house without my phone and I’d really like to be able to have my audiobooks with me.

**Comment by LeoKlaus** [comment]
I've actually got a good chunk of plappa already working on the watch, but downloads were completely broken (which kind of defeats the point).

I'm currently set on fixing a few remaining bugs but will certainly continue work on the Watch app.

**Reply by rtmlp** [reply]
Thanks a lot for this great app. I'm eagerly waiting for watch support. Only thing stopping me from using this everyday as i listen to lot of audiobooks while running

**Reply by LeoKlaus** [reply]
The Watch app is currently in TestFlight. If you want to try it, you can join here: https://testflight.apple.com/join/oSAekRD5

It should release in the AppStore early next year.

**Reply by rtmlp** [reply]
Wow. I just installed the app through test flight. Can't believe I am able to see a progress bar for the download on Apple watch. I will get the premium app now :D
